pinot-controller/src/main/resources/app/components/Homepage/Operations/RebalanceServer/RebalanceServerConfigurationOptions/RebalanceServerConfigurationOptionInteger.tsx

```diff
@@ -0,0 +1,39 @@
+import {Box, FormControl, TextField, Typography} from "@material-ui/core";
```

Comment:
The linter is failing due to the missing headers in all these new files. You should be able to fix this by running `mvn license:format` or else copying the license headers from other files manually.

Reply:
updated

Reply:
it's passing now